flow360/component/flow360_params/flow360_params.py

```diff
@@ -1323,3 +1328,8 @@ def __init__(self, filename: str = None, **kwargs):
     # pylint: disable=missing-class-docstring,too-few-public-methods
     class Config(Flow360BaseModel.Config):
         extra = "allow"
+
+class UnitDefaults(Flow360BaseModel):
+    defaults_SI: Dict = pd.Field(alias="defaultsSI", default=SI_unit_system.defaults())
+    defaults_CGS: Dict = pd.Field(alias="defaultsCGS", default=CGS_unit_system.defaults())
+    defaults_imperial: Dict = pd.Field(alias="defaultsImperial", default=imperial_unit_system.defaults())
```

Comment:
put in `ROOT/tools/integrations/unit_systems_defaults.py`

Reply:
Fixed in bfd5375